src/engine/events/MouseClickEvent.hpp

```diff
@@ -1,10 +1,13 @@
 #pragma once
 
-#include "EventArgs.hpp"
+#include <cstdint>
 
 #include <SDL_events.h>
 #include <SDL_mouse.h>
-#include <cstdint>
+
+#include "events/EventArgs.hpp"
+
+#define SDLK_IGNORE SDLK_CALCULATOR
```

Comment:
unused?

Reply:
No, see comments for `Engine::AddMenu`, but we'll discuss that later as well.